whipper/common/program.py

```diff
@@ -614,3 +617,13 @@ def writeLog(self, discName, logger):
         self.logPath = logPath
 
         return logPath
+
+    def move_tocfile(self, discName):
+        """Move cdrdao's tocfile to the discname path (.toc ending)"""
```

Comment:
This could be a little more clear. Maybe "to the output directory ('DISCNAME.toc')", "... ('[discname].toc')", etc.